disco_aws_automation/disco_rds.py

```diff
@@ -409,12 +413,19 @@ def modify_db_parameter_group(self, db_parameter_group_name, parameters):
                                                                'MinimumEngineVersion': 'someversion',
                                                                'ApplyMethod': 'pending-reboot'}])
 
-    def recreate_db_parameter_group(self, database_name, db_parameter_group_name,
+    def recreate_db_parameter_group(self, env_name, database_name, db_parameter_group_name,
                                     db_parameter_group_family):
         """
         Check if there are any custom parameters for this instance
         Custom Parameters are set in ./rds/engine_specific/{instance_identifier}.ini
         If this file doesn't exist, we'll use default RDS parameters
+        Args:
+            env_name (str): The environment name to use when reading the config file.
+                            This is only used in the section name of the config file.
+                            The parameter group is always created for the current environment
+            database_name:
```

Comment:
I can add one but wasn't sure if it was worth it since the argument name was already pretty descriptive